### PR TITLE
Ward

### PR DESCRIPTION
**Pull request by bioye**
Introduced Wards listing, bare bones polling unit page and bare bones ward page

### CODE REVIEW

File: src/main/resources/templates/ward.html [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+    <script type="text/javascript" th:src="@{/js/googleanalytics.js}"></script>
+	</head>
+    <!--Start of Tawk.to Script-->
+    <script type="text/javascript" th:src="@{/js/tawkto.js}"></script>
+    <!--End of Tawk.to Script-->
+	<body>
+
+	<hr />
+
+	<h4>Ward</h4>
```
Comment (sfagade) [inline src/main/resources/templates/ward.html]: Same as other HTML

File: src/main/java/com/naijagov/naijagov/model/PollingCentre.java [hunk 2]
```diff
@@ -5,7 +5,7 @@
 import javax.persistence.Table;
 
 @Entity
-@Table(name="polling_unit")
+//@Table(name="polling_unit")
```
[the remark this answers is not in the document]
Comment (bioye) [reply src/main/java/com/naijagov/naijagov/model/PollingCentre.java]: Fixed @sfagade

File: src/main/java/com/naijagov/naijagov/model/Ward.java [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+import java.io.Serializable;
+
+import javax.persistence.Entity;
+import javax.persistence.Id;
+import javax.persistence.ManyToOne;
+
+@Entity
+public class Ward implements Serializable{
+	
+	/*
```
[the remark this answers is not in the document]
Comment (bioye) [reply src/main/java/com/naijagov/naijagov/model/Ward.java]: Fixed @sfagade

File: src/main/resources/templates/pollingunit.html [hunk 4]
```diff
@@ -0,0 +1,35 @@
+<!DOCTYPE html>
+<html xmlns="http://www.w3.org/1999/xhtml"
+      xmlns:th="http://www.thymeleaf.org">
+<head>
```
[the remark this answers is not in the document]
Comment (bioye) [reply src/main/resources/templates/pollingunit.html]: Fixed in latest commit. @sfagade

File: src/main/resources/templates/pollingunitsthyme.html [hunk 5]
```diff
@@ -24,14 +24,18 @@ <h2>Polling Units</h2>
     <table class="striped responsive-table">
         <tr>
             <th>#</th>
-            <th><a th:href="@{?sortBy=fullCode&sortDirection=ASC}">Code</th>
-            <th><a th:href="@{?sortBy=description&sortDirection=ASC}">Description</th>
-            <th><a th:href="@{?sortBy=longitude&sortDirection=ASC}">Longitude</th>
-            <th><a th:href="@{?sortBy=latitude&sortDirection=ASC}">Latitude</th>
+            <th><a th:href="@{?sortBy=fullCode&sortDirection=ASC}">Code</a></th>
+            <th><a th:href="@{?sortBy=description&sortDirection=ASC}">Description</a</th>
```
[the remark this answers is not in the document]
Comment (bioye) [reply src/main/resources/templates/pollingunitsthyme.html]: Fixed in latest commit.  Kindly re-review @sfagade

File: src/main/resources/templates/ward.html (hunk 1 above)
Comment (bioye) [reply src/main/resources/templates/ward.html]: Fixed in latest commit.  Kindly re-review @sfagade

File: src/main/java/com/naijagov/naijagov/model/PollingUnit.java [hunk 6]
```diff
@@ -30,11 +37,16 @@ public int getId() {
         return id;
     }
 
+	@ManyToOne
```
[the remark this answers is not in the document]
Comment (bioye) [reply src/main/java/com/naijagov/naijagov/model/PollingUnit.java]: Fixed in latest comment.  Please review.  Thank you.  @sfagade

File: src/main/resources/templates/pollingunitsthyme.html [hunk 7]
```diff
@@ -1,59 +1,68 @@
...
-            <a th:href="@{/pollingunitsthyme(page=${form.pollingUnitsPage.number-1},size=${form.pollingUnitsPage.size})}">Previous</a>
-        </span>
-        Page<span th:text="${form.pollingUnitsPage.number+1}"></span> of <span th:text="${form.pollingUnitsPage.totalPages}"></span>
-        <span th:if="${form.pollingUnitsPage.hasNext()}">
-            <a th:href="@{/pollingunitsthyme(page=${form.pollingUnitsPage.number+1},size=${form.pollingUnitsPage.size})}">Next</a>
-        </span>
-        <a th:href="@{/pollingunitsthyme(page=${form.pollingUnitsPage.totalPages-1},size=${form.pollingUnitsPage.size})}">Last</a>
+      <a th:href="@{/pollingunitsthyme(page=0,size=${form.pollingUnitsPage.size})}">First</a>
+      <span th:if="${form.pollingUnitsPage.hasPrevious()}">
+        <a
```
Comment (sfagade) [inline src/main/resources/templates/pollingunitsthyme.html]: This formatting looks somehow

File: src/main/java/com/naijagov/naijagov/model/PollingCentre.java [hunk 8]
```diff
@@ -36,7 +34,7 @@ public int getId() {
 	private int code;
```
Comment (sfagade) [inline src/main/java/com/naijagov/naijagov/model/PollingCentre.java]: This formatting is out of place

File: src/main/resources/templates/pollingunitsreact.html [hunk 9]
```diff
@@ -1,31 +1,33 @@
...
+
+  <link rel="stylesheet" type="text/css" th:href="@{/css/style.css}" />
+
+<body>
+  <h2>Polling Units</h2>
+  <div id="react">
+    <!-- Global site tag (gtag.js) - Google Analytics -->
+    <script async src="https://www.googletagmanager.com/gtag/js?id=UA-141725675-1"></script>
+    <script type="text/javascript" th:src="@{/js/googleanalytics.js}"></script>
+    </head>
```
[the remark this answers is not in the document]
Comment (bioye) [reply src/main/resources/templates/pollingunitsreact.html]: Fixed.  Kindly review.

File: src/main/resources/templates/pollingunitsthyme.html (hunk 7 above)
Comment (bioye) [reply src/main/resources/templates/pollingunitsthyme.html]: Fixed.  Kindly review @sfagade

File: src/main/java/com/naijagov/naijagov/model/PollingCentre.java (hunk 8 above)
Comment (bioye) [reply src/main/java/com/naijagov/naijagov/model/PollingCentre.java]: Fixed.  Kindly review @sfagade

**Comment by bioye** [comment]
> I will suggest you use some kind of UI / frontend templating tool. The current state of your HTMLs will be problematic as your project phase progresses

@sfagade, thanks for the quality feedback and suggestions.  What UI / frontend templating tools would you recommend?

**Pull request merged**